src/modules/acbs/dto/acbs-get-facility-loan-response.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { DateString } from '@ukef/helpers';
+
+export type AcbsGetFacilityLoanResponseDto = AcbsGetFacilityLoanResponseItem[];
+
+export interface AcbsGetFacilityLoanResponseItem {
+  PortfolioIdentifier: string;
+  LoanIdentifier: string;
+  ParentFacilityIdentifier: string;
+  PrimaryParty: {
+    PartyIdentifier: string;
+  };
+  Currency: {
+    CurrencyCode: string;
+  };
+  PrincipalBalance: number;
+  InterestBalance: number;
+  FeeBalance: number;
+  OtherBalance: number;
+  DiscountedPrincipal: number;
+  EffectiveDate: DateString;
+  MaturityDate: DateString;
+  ProductGroupCode: string; // TODO APIM-126: deprecated as of 12-01-24 (use ProductGroup.ProductGroupCode instead), but will API be retired by then anyway?
```

Comment:
ACBS documentation might use US date format. If 12-01-24 is December 2024, we are hoping to replace ACBS by then, but it is not 100%.

Could we change to ProductGroup.ProductGroupCode to have less TODO items.

Reply:
Changed.